packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,377 @@
+"displayName": "Display Name for UI",
+"name": "aNewType"
+}'
+```
+
+#### Response Example
+
+```json
+{
+  "_links": {
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "schema"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2019-04-10T02:00:01.000Z",
+  "createdBy": "00ufnlhzppWItClAI0g4",
+  "default": false,
+  "description": "Any description that means something useful to you",
+  "displayName": "Display Name for UI",
+  "id": "otyfnly5cQjJT9PnR0g4",
+  "lastUpdated": "2019-04-10T02:00:01.000Z",
+  "lastUpdatedBy": "00ufnlhzppWItClAI0g4",
+  "name": "aNewType",
+  "ref": {
+    "id": "otyfnly5cQjJT9PnR0g4",
+    "refClass": "com.saasure.db.dto.platform.entity.cvd.CVDType"
+  }
+}
+```
+
+### Update User Type
+
+<ApiOperation method="PUT" url="/api/v1/meta/types/user/{typeId}" />
+
+<ApiOperation method="POST" url="/api/v1/meta/types/user/{typeId}" />
+
+Updates an existing User Type. A PUT is a full replace operation; a POST is a partial update. The schema associated with this type is not editable with this API. If you want to edit the schema, use the [schema update API](/docs/reference/api/schemas/#add-property-to-user-profile-schema).
+
+#### Request Parameters
+
+The [User Type](#user-type-model) you wish to update
+
+#### Response parameters
+
+The updated [User types](#user-type-model)
+
+#### Request Example
+
+```bash
+curl -s -XPUT -H "Content-Type: application/json" -H "Authorization: SSWS ${api_token}" https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4 -d '{
+"description": "Updated description",
+"displayName": "Updated Name for UI",
+"name": "updatedTypeName"
+}'
+```
+
+#### Response Example
+
+```json
+{
+  "_links": {
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "schema"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2019-04-10T02:00:01.000Z",
+  "createdBy": "00ufnlhzppWItClAI0g4",
+  "default": false,
+  "description": "Any description that means something useful to you",
+  "displayName": "TheDisplayName",
+  "id": "otyfnly5cQjJT9PnR0g4",
+  "lastUpdated": "2019-04-10T02:00:01.000Z",
+  "lastUpdatedBy": "00ufnlhzppWItClAI0g4",
+  "name": "aNewType",
+  "ref": {
+    "id": "otyfnly5cQjJT9PnR0g4",
+    "refClass": "com.saasure.db.dto.platform.entity.cvd.CVDType"
+  }
+}
+```
+
+### Delete User Type
+
+This operation is not supported for beta. It is on the roadmap to be implemented before the User Types feature becomes Generally Available.
+
+## User Schema Operation Extensions to Non-Default Types
+
```

Comment:
Rather than put this information on this page, would it make sense to instead update User Schema page?

Reply:
The section this was discussing has in fact been removed completely, with the contents folded into the Schemas API page.